app/helpers/application_helper.rb

```diff
@@ -7,7 +7,8 @@ def autolink_tags(text)
 
   def nav_path(link_text, link_path, http_method=nil)
     css_class = "navigation"
-    css_class << " current" if current_page?(link_path)
+    css_class << " current" if current_page?(link_path) ||
+        (current_page?(mileage_entry_path) && link_text == t("titles.projects.index"))
```

Comment:
Line is too long. [86/80]<br>Align the operands of a condition in an `if` statement spanning multiple lines.